superset/assets/javascripts/explore/stores/visTypes.js

```diff
@@ -1,18 +1,22 @@
 import { D3_TIME_FORMAT_OPTIONS } from './controls';
 import * as v from '../validators';
+import { t } from '../../locales';
 
 export const sections = {
   druidTimeSeries: {
-    label: 'Time',
+    label: ('Time'),
+    labelDisplay: t('Time'),
```

Comment:
I hate the fact that `labelDisplay` and `descriptionDisplay` have a negative impact on our app's memory footprint despite the fact that it's only used for flask-babel to find the strings to translate. What about using a noop `t` function in this context?

From what I understand, you could simply go `const t = () => {};` instead of `import { t } from '../../locales';` and babel should still pick up all the calls. Then you can get rid of all the `*Display` entries. Wouldn't that work? If it works, we should do that and make sure to add a comment that explains the problematic of dealing with i18n strings in module scope. The comment should read something like:

```
For i18n strings that are defined in module scope, we cannot have the guarantee that the translation dictionary hasn't been initialized at module-import-time. In those cases we use a noop `t()` function so that the strings can get detected by Python's babel, but those strings need to be translated at runtime where they are used as variables.
```

Reply:
removed all labelDisplay and descriptionDisplay.
created a new module, i18n, to parse and create Jed object. since i18n is dependency of locales.jsx, these modules will be loaded before vizType, control.jsx etc.